features/signal/basic/feature.py

```diff
@@ -0,0 +1,30 @@
+from datetime import timedelta
+
+from temporalio import workflow
+from temporalio.client import WorkflowHandle
+
+from harness.python.feature import Runner, register_feature
+
+WORKFLOW_INITIAL_STATE = ""
+SIGNAL_ARG = "signal-arg"
+
+
+@workflow.defn
+class Workflow:
+    def __init__(self) -> None:
+        self._state = WORKFLOW_INITIAL_STATE
+
+    @workflow.run
+    async def run(self) -> str:
+        await workflow.wait_condition(lambda: self._state != WORKFLOW_INITIAL_STATE)
+        return f"{input} {self._state}"
+
+    @workflow.signal
+    async def my_signal(self, arg: str):
+        self._state = arg
+
+
+register_feature(
+    workflows=[Workflow],
+    expect_run_result=SIGNAL_ARG,
```

Comment:
This doesn't seem to match the result. Also, may be missing calls to actually send a signal.

Reply:
Oops, I pushed a corrupted version without running all features again. Fixed.